modules/claims_api/lib/claims_api/error/soap_error_handler.rb

```diff
@@ -35,6 +35,10 @@ def get_exception
         raise ::Common::Exceptions::UnprocessableEntity.new(
           detail: 'Please try again after checking your input values.'
         )
+      elsif participant_has_open_claims?
+        raise ::Common::Exceptions::ServiceError.new(
+          detail: 'PtcpntIdA has open claims.'
+        )
```

Comment:
works for now, but as we handle more errors, it might make sense for each service to check for their respective errors. with centralized error handling, each bgs call will check for all errors, even those unrelated to their service.

Reply:
[API-41059](https://jira.devops.va.gov/browse/API-41059)